src/extractor/restriction_parser.cpp

```diff
@@ -111,21 +113,26 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
         else if (value.find("no_") == 0 && !boost::algorithm::ends_with(value, "_on_red"))
         {
             is_only_restriction = false;
+            if (value.find("no_exit") == 0)
```

Comment:
nit. 
can we use [`boost::algorithm::starts_with`](https://www.boost.org/doc/libs/1_41_0/doc/html/boost/algorithm/starts_with.html) instead? Should be a bit faster.